.github/workflows/dashboard.yaml

```diff
@@ -1,11 +1,7 @@
 name: 'Dashboard'
 
 on:
-  push:
```

Comment:
I think we can keep this workflow when pushing to `main`.

Reply:
we can, but the workflow would run three times for the same commit:
- in the PR (check before merge)
- in the changeset workflow (check before release)
- in the dashboard workflow